### PR TITLE
Minor documentation updates

### PR DESCRIPTION
**Pull request by purefunctor**
Just a few documentation fixes I came across while exploring the library.

### CODE REVIEW

File: src/Data/Veither.purs [hunk 1]
```diff
@@ -268,8 +268,8 @@ vsafe (Veither v) = on _veither identity case_ v
 -- | vhandle _b show bar == ((pure "3") :: Veither () String)
 -- | vhandle _b show foo == ((pure "2") :: Veither () String)
 -- |
--- | safe (vhandle _b show bar) == "3"
--- | safe (vhandle _b show foo) == "2"
+-- | vsafe (vhandle _b show bar) == "3"
+-- | vsafe (vhandle _b show foo) == "2"
```
Comment (JordanMartinez) [inline src/Data/Veither.purs:272]: This change makes sense

File: README.md [hunk 2]
```diff
@@ -66,11 +66,11 @@ foo = do
 
 main :: Effect Unit
 main = do
-  -- if you would use `fromRight` (or some other
+  -- if you would use `either` (or some other
   -- `Either` function) for `Either`, 
   -- add a 'v' in front of it and you have the
   -- corresponding function name for `Veither`
-  log $ vfromRight handleFailure foo 
+  log $ veither handleFailure identity foo
```
Comment (JordanMartinez) [inline README.md:73]: But why was this change made?

Comment (purefunctor) [reply README.md:73]: This doesn't seem to be the right usage for `vfromRight`, is it?

Comment (JordanMartinez) [reply README.md:73]: Ah, I see now. Yeah, I'm using it wrong here.

**Comment by JordanMartinez** [comment]
Thanks for the PR and answering my questions!

**Pull request merged**